### PR TITLE
Merchant/items/us 10

### PR DESCRIPTION
**Pull request by zkevkev**
This branch contains functionality for US10, listed below.

As a merchant,
When I visit my merchant items index page
Then I see two sections, one for "Enabled Items" and one for "Disabled Items"
And I see that each Item is listed in the appropriate section

### CODE REVIEW

File: app/models/merchant.rb [hunk 1]
```diff
@@ -30,4 +30,12 @@ def top_five_customers_count
     .limit(5)
     .count("transactions.id")
   end
+
+  def enabled_items
```
Comment (kameronk92) [inline app/models/merchant.rb]: Nice work putting these in the model

File: app/views/merchants/items/index.html.erb [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-
-    <% if item.status == "enabled" %>
-      Enabled  
-      <%= button_to "Disable Item", merchant_item_status_path(@merchant, item), method: :patch, params: { status: "disabled" }, data: { turbo: false }, local: true %>
-    <% end %>
-  </section>
-<% end %>
\ No newline at end of file
+<section id="disabled-items">
+  <h3>Enabled Items</h3>
```
Comment (kameronk92) [inline app/views/merchants/items/index.html.erb]: I think your headers may be mixed up, between enabled/ disabled items

File: spec/features/merchants/items/index_spec.rb [hunk 3]
```diff
@@ -70,8 +71,33 @@
 
     expect(current_path).to eq(merchant_items_path(@merchant_1))
     within("##{@item_1.id}") do
-      expect(page).to have_content("Disabled")
       expect(page).to have_button("Enable Item")
     end
   end
+
+  # US10
+  it "enabled items are listen in a separate section" do
```
Comment (kameronk92) [inline spec/features/merchants/items/index_spec.rb]: spelling - listed?

**Review by kameronk92 — CHANGES_REQUESTED** [review]

File: app/views/merchants/items/index.html.erb (hunk 2 above)
Comment (zkevkev) [reply app/views/merchants/items/index.html.erb]: Good catch! Just updated those headers.

File: spec/features/merchants/items/index_spec.rb (hunk 3 above)
Comment (zkevkev) [reply spec/features/merchants/items/index_spec.rb]: Oops, just fixed.

Comment (kameronk92) [reply spec/features/merchants/items/index_spec.rb]: Thanks for fixing!

**Pull request merged**